modules/programs/broot.nix

```diff
@@ -44,233 +180,50 @@ in {
-          permissions_fg = "grayscale(12)";
-          size_bar_full_bg = "red";
-          size_bar_void_bg = "black";
-          directory_fg = "lightyellow";
-          input_fg = "cyan";
-          flag_value_fg = "lightyellow";
-          table_border_fg = "red";
-          code_fg = "lightyellow";
-        }
-      '';
-      description = ''
-        Color configuration.
-        </para><para>
-        Complete list of keys (expected to change before the v1 of broot):
-
-        <itemizedlist>
-          <listitem><para><literal>char_match</literal></para></listitem>
-          <listitem><para><literal>code</literal></para></listitem>
-          <listitem><para><literal>directory</literal></para></listitem>
-          <listitem><para><literal>exe</literal></para></listitem>
-          <listitem><para><literal>file</literal></para></listitem>
-          <listitem><para><literal>file_error</literal></para></listitem>
-          <listitem><para><literal>flag_label</literal></para></listitem>
-          <listitem><para><literal>flag_value</literal></para></listitem>
-          <listitem><para><literal>input</literal></para></listitem>
-          <listitem><para><literal>link</literal></para></listitem>
-          <listitem><para><literal>permissions</literal></para></listitem>
-          <listitem><para><literal>selected_line</literal></para></listitem>
-          <listitem><para><literal>size_bar_full</literal></para></listitem>
-          <listitem><para><literal>size_bar_void</literal></para></listitem>
-          <listitem><para><literal>size_text</literal></para></listitem>
-          <listitem><para><literal>spinner</literal></para></listitem>
-          <listitem><para><literal>status_error</literal></para></listitem>
-          <listitem><para><literal>status_normal</literal></para></listitem>
-          <listitem><para><literal>table_border</literal></para></listitem>
-          <listitem><para><literal>tree</literal></para></listitem>
-          <listitem><para><literal>unlisted</literal></para></listitem>
-        </itemizedlist></para>
-
-        <para>
-        Add <literal>_fg</literal> for a foreground color and
-        <literal>_bg</literal> for a background colors.
-      '';
+      description = "Verbatim config entries";
     };
   };
 
   config = mkIf cfg.enable {
     home.packages = [ cfg.package ];
 
     xdg.configFile."broot/conf.toml".source =
-      tomlFormat.generate "broot-config" brootConf;
+      tomlFormat.generate "broot-config" cfg.config;
 
     # Dummy file to prevent broot from trying to reinstall itself
     xdg.configFile."broot/launcher/installed-v1".text = "";
 
+    programs.broot.config = builtins.fromTOML (builtins.readFile
+      (pkgs.runCommand "default-conf.toml" {
+        nativeBuildInputs = with pkgs; [ hjson remarshal ];
+      } ''
+        hjson -c ${cfg.package.src}/resources/default-conf.hjson \
+        | remarshal --input-format json --output-format toml --output $out
+      ''));
+
```

Comment:
This used IFD (import-from-derivation) and will cause problems for some users and some workflows.

An alternative way is to override the toml generation above and merge the two documents if possible.

Reply:
Resolving with regards to [decision](https://github.com/nix-community/home-manager/pull/2644#issuecomment-1166668873)